docs/devsguide/new_dev_guide.rst

```diff
@@ -92,15 +35,11 @@ developers, it is recommended that you install PyNE from the source.
 
 Make sure that your device has all of the dependencies downloaded. You 
 can check this by typing the name of the program into your command line 
-followed by "-v" (e.g. $ python -v). If you are just starting as a 
+followed by "--version" (e.g., $ python --version). If you are starting as a
```

Comment:
can you surround these with `` `x` ``  to render as code?

Reply:
I tried it on my version of the website, and it rendered as italics. Is that how it's supposed to look?

Reply:
I think the --version in quotes would render as italicized? but $ python --version can have double ` around it to render as monospaced

Reply:
the same that single ` does in this comment: `$ python --version`

Reply:
Interesting. On github, the quotes don't do anything (because it's either single * or double ** that does italics).

Reply:
I think I figured it out, because its sphinx and the theme it requires :code: before something in `` will render in the same way as in markdown.

Reply:
I'll rebase and include this addition in the push at the end